src/main/java/com/example/shopapp/services/CartService.java

```diff
@@ -0,0 +1,11 @@
+package com.example.shopapp.services;
+
+import com.example.shopapp.dto.CartDTO;
+import com.example.shopapp.dto.ProductDTO;
+
+import java.util.List;
+
+public interface CartService {
```

Comment:
This and all the service interfaces have just one implementation each. It doesn't really make sense to have those interfaces.